src/screens/Topics/index.js

```diff
@@ -316,21 +278,22 @@ const styles = StyleSheet.create({
     fontFamily: 'Inter',
     fontStyle: 'normal',
     fontWeight: '400',
-    fontSize: 10,
+    fontSize: normalizeFontSize(10),
     textAlign: 'center',
     color: colors.blackgrey
   },
   button: {
-    backgroundColor: colors.gray
+    backgroundColor: colors.gray,
+    borderRadius: dimen.normalizeDimen(8)
   },
   scrollButtonParent: {
-    paddingHorizontal: 22
+    paddingHorizontal: dimen.normalizeDimen(22)
   },
   containerContent: {
-    paddingRight: 20
+    paddingRight: dimen.normalizeDimen(20)
   },
   reloadTopicButtonContainer: {
-    marginTop: 60
+    marginTop: dimen.normalizeDimen(60)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application. The use of `dimen.normalizeDimen` function to normalize dimensions is a good practice in React Native to ensure consistent layout across different screen sizes and resolutions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->